project03.py

```diff
@@ -233,12 +233,18 @@ def verifyDeathBefore150YearsOld(person):
     years_in_seconds = 150 * 365 * 24 * 60 * 60
     return age <= years_in_seconds
   
-def marriageAfter14(person1, person2):
-    #check if both people are above 14 years of age to marry
-    if (person1['age'] > 14 and person2['age'] > 14):
-        return True
-    else:
-        return False
+def verifyMarriageAfter14(family):
+    if not family['divorced'] == 'NA':
+        wife = individualsDict[family['wifeId']]
+        husband = individualsDict[family['husbandId']]
+        
+        #if both wife and husband are over 14 years of age
+        if (wife['age'] > 14 and husband['age'] > 14):
```

Comment:
Not correct, must be 14 years after birth, not currently 14 years of age

Reply:
To be clear, print out a helpful error message every time:
- A families marriage date minus the wife's birthday was less than 14 years (unix time)
- A familieis marriage date minus the husbamd's birthday was less than 14 years (unix time)